### PR TITLE
Remove redundant team logic that is now done in the API

### LINKED ISSUE #363: AP: Remove redundant/unused/defunct code used to provide team functionality
The "use team app data" checkbox functionality will be removed, which means that any pages associated with it will be inaccessible to end users, making that code obsolete.

The current clutter caused by old, redundant code has caused confusion among current developers and may cause technical debt for future developers as well. Therefore, this issue has been raised to address code that is not needed anymore.

This will make issues #257 and #263 significantly easier. It will also stop developers from editing code that will not have any effect on the end product.

### PR DESCRIPTION
**Pull request by Jayden876212**
Fixes #363

### CODE REVIEW

**Review by jaimielovett — CHANGES_REQUESTED** [review]

File: ap_src/ap_app/absences.py [hunk 1]
```diff
@@ -6,31 +6,20 @@
 #edit_recurring_absences
 #add_recurring
```
[the remark this answers is not in the document]
Comment (Jayden876212) [reply ap_src/ap_app/absences.py]: These were fixed with commit 1d311b54d0bec57dfbc6b3e00c1182de76a94932

File: ap_src/ap_app/absences.py [hunk 2]
```diff
@@ -304,4 +293,60 @@ def add_recurring(request) -> render:
         )
 
     content = {"form": form, "form2": form2}
-    return render(request, "ap_app/add_recurring_absence.html", content)
\ No newline at end of file
+    return render(request, "ap_app/add_recurring_absence.html", content)
+
+#JC - Add an absence via the form
```
[the remark this answers is not in the document]
Comment (Jayden876212) [reply ap_src/ap_app/absences.py]: I have removed them with commit 9044bee, which improved other aspects of the comments (see description of commit).

**Comment by Jayden876212** [comment]
Issue #371 should also be resolved by the merging of this PR

**Comment by Jayden876212** [comment]
This also fixes #372 through refactoring the source files

**Review by jaimielovett — APPROVED** [review]

**Pull request merged**